corehq/pillows/mappings/app_mapping.py

```diff
@@ -67,6 +67,7 @@
                            'type': 'multi_field'},
                 'force_http': {'type': 'boolean'},
                 'is_released': {'type': 'boolean'},
+                'is_auto_generated': {'type': 'boolean'},
```

Comment:
`is_auto_generated ` is going to be missing from the couch doc json for all existing docs. will that make this value be missing from es after reindex, or will the reindex honor the default of False?

Reply:
It should honor the default - it's the same setup as the vellum_case_management property added to apps when we migrated to case management in the form builder. I'm currently testing on staging, should find out if there are issues.